carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/decorator/ExtendedWebElement.java

```diff
@@ -1192,6 +1192,10 @@ public ExtendedWebElement format(Object... objects) {
             LOGGER.debug("Base64 image representation has benn successfully obtained after formatting.");
             by = MobileBy.image(base64image);
         }
+
+        if (locator.startsWith("By.AndroidUIAutomator: ")) {
+            by = MobileBy.AndroidUIAutomator(String.format(StringUtils.remove(locator, "By.AndroidUIAutomator: "), objects));
```

Comment:
let's split formattedLocator declaration and log it's value on debug layer as we did for By.Image (line 1182, 1184)